builder/scaleway/builder.go

```diff
@@ -19,6 +21,8 @@ import (
 // BuilderId is the unique id for the builder
 const BuilderId = "hashicorp.scaleway"
 
+var nightly = flag.Bool("nightly", os.Getenv("PACKER_ACC") == "1", "Run nightly tests")
```

Comment:
Run acceptance test. The fact that they are nightly is a detail that could be changed. Acceptance tests (e2e tests) is what they are.